plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerTypeMapping.java

```diff
@@ -804,6 +805,47 @@ public void testSqlServerDatetimeOffset(ZoneId sessionZone)
                 .execute(getQueryRunner(), session, sqlServerCreateAndInsert("test_sqlserver_datetimeoffset"));
     }
 
+    @Test
+    public void testSqlServerUniqueIdentifier()
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("uniqueidentifier", "NULL", UUID, "CAST(NULL AS uuid)")
+                .addRoundTrip("uniqueidentifier", "'d5416a8a-5faf-208d-a0cc-aaf97732f49a'", UUID, "uuid 'd5416a8a-5faf-208d-a0cc-aaf97732f49a'")
+                .addRoundTrip("uniqueidentifier", "'00000000-0000-0000-0000-000000000000'", UUID, "uuid '00000000-0000-0000-0000-000000000000'")
+                .addRoundTrip("uniqueidentifier", "'ffffffff-ffff-ffff-ffff-ffffffffffff'", UUID, "uuid 'ffffffff-ffff-ffff-ffff-ffffffffffff'")
+                .addRoundTrip("uniqueidentifier", "'0E984725-C51C-4BF4-9960-E1C80E27ABA0wrong'", UUID, "uuid '0E984725-C51C-4BF4-9960-E1C80E27ABA0'")
+                .execute(getQueryRunner(), sqlServerCreateAndInsert("test_sqlserver_uniqueidentifier"));
+
+        SqlDataTypeTest.create()
+                .addRoundTrip("uniqueidentifier", "NULL", UUID, "CAST(NULL AS uuid)")
+                .addRoundTrip("uniqueidentifier", "uuid 'd5416a8a-5faf-208d-a0cc-aaf97732f49a'", UUID, "uuid 'd5416a8a-5faf-208d-a0cc-aaf97732f49a'")
+                .addRoundTrip("uniqueidentifier", "uuid '00000000-0000-0000-0000-000000000000'", UUID, "uuid '00000000-0000-0000-0000-000000000000'")
+                .addRoundTrip("uniqueidentifier", "uuid 'ffffffff-ffff-ffff-ffff-ffffffffffff'", UUID, "uuid 'ffffffff-ffff-ffff-ffff-ffffffffffff'")
+                .execute(getQueryRunner(), sqlServerCreateAndTrinoInsert("test_sqlserver_uniqueidentifier"));
```

Comment:
Make the `uuid` in the **Trino literal upper case** to keep the code style consistent.

Suggested changes:
```
        SqlDataTypeTest.create()
                .addRoundTrip("uniqueidentifier", "NULL", UUID, "CAST(NULL AS UUID)")
                .addRoundTrip("uniqueidentifier", "'d5416a8a-5faf-208d-a0cc-aaf97732f49a'", UUID, "UUID 'd5416a8a-5faf-208d-a0cc-aaf97732f49a'")
                .addRoundTrip("uniqueidentifier", "'00000000-0000-0000-0000-000000000000'", UUID, "UUID '00000000-0000-0000-0000-000000000000'")
                .addRoundTrip("uniqueidentifier", "'ffffffff-ffff-ffff-ffff-ffffffffffff'", UUID, "UUID 'ffffffff-ffff-ffff-ffff-ffffffffffff'")
                .addRoundTrip("uniqueidentifier", "'0E984725-C51C-4BF4-9960-E1C80E27ABA0wrong'", UUID, "UUID '0E984725-C51C-4BF4-9960-E1C80E27ABA0'")
                .execute(getQueryRunner(), sqlServerCreateAndInsert("test_sqlserver_uniqueidentifier"));

        SqlDataTypeTest.create()
                .addRoundTrip("uniqueidentifier", "NULL", UUID, "CAST(NULL AS UUID)")
                .addRoundTrip("uniqueidentifier", "UUID 'd5416a8a-5faf-208d-a0cc-aaf97732f49a'", UUID, "UUID 'd5416a8a-5faf-208d-a0cc-aaf97732f49a'")
                .addRoundTrip("uniqueidentifier", "UUID '00000000-0000-0000-0000-000000000000'", UUID, "UUID '00000000-0000-0000-0000-000000000000'")
                .addRoundTrip("uniqueidentifier", "UUID 'ffffffff-ffff-ffff-ffff-ffffffffffff'", UUID, "UUID 'ffffffff-ffff-ffff-ffff-ffffffffffff'")
                .execute(getQueryRunner(), sqlServerCreateAndTrinoInsert("test_sqlserver_uniqueidentifier"));
```

Reply:
There's no need to upeprcase `uuid` in my opinion.

Reply:
Okay, follow what @ebyhr says, no change needed, thanks for the explanation.